### PR TITLE
feature: add `qux` interface

### PR DESCRIPTION
**Pull request by dmkonq**
This PR proposes addtion of interface for function `qux`.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -14,5 +14,9 @@ func baz() (string, error) {
 	return "foo", nil
 }
 
+func qux() (string, error) {
```
Comment (dmkonq) [inline main.go:17]: Why qux? I'd rather use `asdf`.

**Review by dmkonq — COMMENTED** [review]
Seems good

**Review by crowdigit — CHANGES_REQUESTED** [review]
Function `"qux"` is supposed to return `"qux"` not `"foo"`. Fix it please.

File: main.go [hunk 2]
```diff
@@ -14,5 +14,9 @@ func baz() (string, error) {
 	return "foo", nil
```
Comment (crowdigit) [inline main.go]: It should return `"qux"`.

**Review by crowdigit — COMMENTED** [review]
Good

**Review by crowdigit — APPROVED** [review]

**Pull request merged**